nntrainer/tensor/tensor.cpp

```diff
@@ -219,8 +228,27 @@ int Tensor::add_i(float const &value) {
   return ML_ERROR_NONE;
 }
 
+Tensor Tensor::add(Tensor const &m, Tensor &out, float const alpha) const {
+  CREATE_IF_EMPTY_DIMS(out, getDim());
+  scopy(length(), getData(), 1, out.getData(), 1);
```

Comment:
do we really need the copy? we can do it in a single call to transform.

Reply:
This will be changed in upcoming PR. Thanks :)